python_src/requirements.txt

```diff
@@ -1,8 +1,80 @@
-beamngpy~=1.18
-matplotlib~=3.3.2
-click~=7.1.2
-numpy~=1.19.2
-shapely~=1.7.1
-scipy~=1.5.2
-descartes~=1.1.0
-deap~=1.3.1
\ No newline at end of file
+appnope @ file:///opt/concourse/worker/volumes/live/6ca6f098-d773-4461-5c91-a24a17435bda/volume/appnope_1606859448531/work
```

Comment:
This file contains references to your local installation and shall not be updated. Please revert the requirements.txt file to its previous version (restore)

Reply:
Hi Dr. @alessiogambi, I brought back the requirements.txt file and added the separated enviroment.yml file for the Conda environment.